examples/sandbox/package.json

```diff
@@ -9,16 +9,19 @@
     "dev": "yarn parcel index.html",
     "clean": "rm -rf dist .cache"
   },
+  "engines": {
+    "node": "14.x"
```

Comment:
I don't think this is accurate. You're 100% certain this doesn't run on older versions of node? I'm running node 12.x and it compiles fine for me.